code/modules/cargo/exports/materials.dm

```diff
@@ -30,43 +30,43 @@
 // Materials. Nothing but plasma is really worth selling. Better leave it all to RnD and sell some plasma instead.
 
 /datum/export/material/bananium
-	cost = 1000
+	cost = 4000
 	material_id = MAT_BANANIUM
 	message = "cm3 of bananium"
 
 /datum/export/material/diamond
-	cost = 500
+	cost = 2000
 	material_id = MAT_DIAMOND
 	message = "cm3 of diamonds"
 
 /datum/export/material/plasma
-	cost = 200
+	cost = 250
 	k_elasticity = 0
 	material_id = MAT_PLASMA
 	message = "cm3 of plasma"
 
 /datum/export/material/uranium
-	cost = 100
+	cost = 150
```

Comment:
Bounty PR changed uranium 400->100, It was previously more valuable than gold and the like, with the justification comment: "Still useful for both power generation and nuclear annihilation", the comment was removed in the bounty PR as well. I believe the reasoning still stands, I'd put uranium somewhere between 200 and 300.